nodes/alarm_server.py

```diff
@@ -224,31 +120,47 @@ def _create_alarm_handlers(self):
             alarm_name = handler.alarm_name
             severity = handler.severity_required
 
-            if alarm_name not in self.alarms:
-                self.alarms[alarm_name] = Alarm.blank(alarm_name)
-            self.alarms[alarm_name].raised = h.initally_raised
+            # Set initial state if necessary (could have already been added while creating metas)
+            if hasattr(h, 'initial_alarm'):
```

Comment:
I like this feature, but I don't think the initial alarm will get sent to nodes based on the code. The alarm server stores this initial alarm internally, but never publishes it (unless through the service call).